src/test/java/Homework17.java

```diff
@@ -0,0 +1,63 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+
+    @Test
+   public void addSongToPlaylist() throws InterruptedException{
+        String expectedSongAddedMessage ="Added 1 song into \"Val.\"";
+//steps
+launchBrowser();
+navigateLoginPage();
+provideEmail("valentyna.bihdash@testpro.io");
+providePassword("TestTest1!");
+clickSubmit();
+searchSongName("take my hand");
+clickViewAll();
+clickFirstSong();
+clickAddButton();
+choosePlaylist();
+
+Thread.sleep(2000);
+
+        Assert.assertEquals(getAddToPlaylistSuccessMSG(),expectedSongAddedMessage);
+    }
+
+    public void searchSongName(String name) throws InterruptedException{
+        WebElement searchField = driver.findElement(By.cssSelector("input[type='search']"));
```

Comment:
Once you've logged in, there is a delay before the homepage fully loads. To ensure that the page has sufficient time to load all its content before initiating a search, please include a `Thread.sleep(2000)` before sending the search parameters. This delay allows the page to load its content completely, ensuring that the search function will locate the desired content effectively.